ldclient/__init__.py

```diff
@@ -247,6 +249,15 @@ def _match_variation(variation, user):
             return True
     return False
 
+def check_uwsgi():
+    if 'uwsgi' in sys.modules:
+        import uwsgi
+        if not uwsgi.opt.get('enable-threads'):
+            logging.warning('The LaunchDarkly client requires the enable-threads option '
+                            'be passed to uWSGI. If enable-threads is not provided, no '
+                            'threads will run and flag data will not be sent to LaunchDarkly. '
```

Comment:
Not just flag request events, but also custom events and identify events, right?